Extensions/hideavatars.js

```diff
@@ -0,0 +1,239 @@
+//* TITLE Hide Avatars **//
+//* VERSION 0.1.0 **//
+//* DESCRIPTION Hides avatars on a per-url basis **//
+//* DEVELOPER dlmarquis **//
+//* FRAME false **//
+//* BETA true **//
+
+XKit.extensions.hideavatars = new Object({
+
+	running: false,
+	
+	blognames: [],
+	
+	run: function() {
+
+		this.running = true;
+
+		XKit.tools.init_css("hideavatars");
+
+		XKit.extensions.hideavatars.load_blogs();
+		
+		XKit.post_listener.add("hide_avatars", XKit.extensions.hideavatars.hide_avatars);
+		XKit.extensions.hideavatars.hide_avatars();
+		
+	},
+	
+	/*frame_run: function() {
+
+		XKit.tools.init_css("hideavatars");
+
+		XKit.extensions.hideavatars.load_blogs();
+		
+	},*/
+	
+	hide_avatars: function() {
+	    
+	    for (var i=0;i<XKit.extensions.hideavatars.blognames.length;i++) {
+	        $(".post_avatar_link").filter(function( index ) {
+	            return $(this).attr("href").slice(7).split(".")[0] == XKit.extensions.hideavatars.blognames[i].title;
```

Comment:
This is technically making functions within a loop and can be fixed by something like
```javascript
XKit.extensions.hideavatars.blognames.forEach(function(blogname) {
  // current loop body with blognames[i] becoming blogname
});
```